src/main/java/com/fasterxml/jackson/dataformat/xml/ser/ToXmlGenerator.java

```diff
@@ -195,8 +197,14 @@ public void initGenerator()  throws IOException
         try {
             if ((_xmlFeatures & Feature.WRITE_XML_1_1.getMask()) != 0) {
                 _xmlWriter.writeStartDocument("UTF-8", "1.1");
+                if (_xmlPrettyPrinter != null) {
+                  _xmlWriter.writeRaw(NEWLINE);
```

Comment:
This may work; my main concern is that it is not supported by Stax implementation included by JDK.
As this would throw an exception (for non-implemented), I'll have to think about this.
As per my notes on bug, there is no reliably way to inject whitespace outside of root element, unless raw writes (which by-pass underlying XML encoder) are supported.